SpatialGDK/Source/SpatialGDKTests/SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.cpp

```diff
@@ -101,6 +101,12 @@ bool FStopDeployment::Update()
 {
 	FLocalDeploymentManager* LocalDeploymentManager = GetLocalDeploymentManager();
 
+	if (LocalDeploymentManager->IsDeploymentStarting())
```

Comment:
Is this just a runby fix?

Reply:
Some tests were having issues after the change which led to this change, more info here:
https://improbableio.atlassian.net/browse/UNR-4464